### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by marshallswain**
Linting wasn’t running because the jshint-style path globs as arguments.

### CODE REVIEW

File: example/app.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
 const feathers = require('feathers');
 const rest = require('feathers-rest');
 const socketio = require('feathers-socketio');
-const primus = require('feathers-primus');
+// const primus = require('feathers-primus');
 const hooks = require('feathers-hooks');
 const memory = require('feathers-memory');
 const bodyParser = require('body-parser');
-const errors = require('feathers-errors');
+// const errors = require('feathers-errors');
```
Comment (marshallswain) [inline example/app.js]: done

File: test/integration/rest.test.js [hunk 2]
```diff
@@ -40,6 +40,7 @@ describe('REST client authentication', () => {
 
   after(done => {
     server.close(done);
```
Comment (ekryski) [inline test/integration/rest.test.js]: done should be being called already.

Comment (marshallswain) [inline test/integration/rest.test.js]: Yeah.  It's just not happening, though, so the tests end up failing.

Comment (ekryski) [inline test/integration/rest.test.js]: Well it used to be. Let's ship this and I'll check it out. I have some client updates to push soon anyway. Just working through them in a project.

**Comment by ekryski** [comment]
:shipit:

**Pull request merged**